yarn-project/bb-prover/src/avm_proving.test.ts

```diff
@@ -223,7 +232,14 @@ const proveAndVerifyAvmTestContract = async (functionName: string, calldata: Fr[
   // First we simulate (though it's not needed in this simple case).
   const simulator = new AvmSimulator(context);
   const avmResult = await simulator.executeBytecode(bytecode);
-  expect(avmResult.reverted).toBe(false);
+
+  if (typeof assertionErrString === 'undefined') {
```

Comment:
`if (assertionErrString === undefined) {`

Reply:
You still need to change this one.

Reply:
Reference to eslint says it must be a string: https://eslint.org/docs/latest/rules/valid-typeof